### PR TITLE
docs: Add Web Component DevTools docs

### PR DESCRIPTION
**Pull request by Matsuuu**
## What I did

1. Added documentation page on the "Guides" portion of open-wc.org to introduce users to [Web Component DevTools](https://chrome.google.com/webstore/detail/web-component-devtools/gdniinfdlmmmjpnhgnkmfpffipenjljo)

### CODE REVIEW

**Comment by Matsuuu** [comment]
Allright this time without extra commits or other dirt :) Sorry for the hassle @Westbrook

File: docs/guides/tools/web-component-devtools.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+The list of extra support libraries will grow as adoption grows
+
+## Download
+
+You can get the Web Component DevTools from the [Chrome Web Store](https://chrome.google.com/webstore/detail/web-component-devtools/gdniinfdlmmmjpnhgnkmfpffipenjljo/related)
+
+### Setting up
+
+A brief video of showcasing and setting up your development environment to get the most out of DevTools can be found [in Youtube](https://youtu.be/D6W5iX3-E9E)
```
Comment (Westbrook) [inline docs/guides/tools/web-component-devtools.md:61]: Should we just embed the video?

**Comment by Matsuuu** [comment]
AFAIK rocket didn't have a built in embed feature so I just added it with the default youtube embed action

**Comment by Matsuuu** [comment]
Would this be merge-ready now?

**Review by bennypowers — APPROVED** [review]

**Pull request merged**